### PR TITLE
sensors/sensor.h: add devno field

### PR DESCRIPTION
**Pull request by raiden00pl**
## Summary

- sensors/sensor.h: add devno field
add devno field so we can select the correct device when we have multiple sensors of the same type on the bus (e.g. SPI)

## Impact
none 

## Testing
ci

### CODE REVIEW

File: include/nuttx/sensors/sensor.h [hunk 1]
```diff
@@ -1031,6 +1031,10 @@ struct sensor_lowerhalf_s
 
   int type;
 
+  /* Device number */
+
+  int devno;
```
Comment (xiaoxiang781216) [inline include/nuttx/sensors/sensor.h:1036]: but nobody use it?

Comment (raiden00pl) [reply include/nuttx/sensors/sensor.h:1036]: I'll add sensors using this field in next PRs

Comment (xiaoxiang781216) [reply include/nuttx/sensors/sensor.h:1036]: since the common code doesn't use it, should we move to individual driver who need it?

Comment (raiden00pl) [reply include/nuttx/sensors/sensor.h:1036]: yes, you are right. It's better to keep it in private sensor data. I'm closing this PR

**Comment by raiden00pl** [comment]
devno will be kept in private sensor data

**Pull request closed without merging**